.github/workflows/database.yaml

```diff
@@ -0,0 +1,136 @@
+name: Database
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *" # 04:00 UTC
+
+jobs:
+  backup-production:
+    name: Backup production
+    runs-on: ubuntu-latest
+
+    environment:
+      name: production_aks
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Get resource group name
+        id: resource-group-name
```

Comment:
Do we need this id? it's not referenced anywhere

Reply:
Ah it's supposed to be used here https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1462/files#diff-8f152a06185ed923f476a31dbd85c48b4de71d9f3ef686a69d1869f667fecb43R56 but I got the name wrong.